plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-payment-page.spec.js

```diff
@@ -110,6 +110,14 @@ test.describe(
 			await test.step( 'Select payment method and pay for the order', async () => {
 				// explicitly select the payment method
 				await page.getByText( 'Direct bank transfer' ).click();
+
+				// Dismiss notice if shown
+				if (
+					page.getByRole( 'link', { name: 'Dismiss' } ).isVisible()
+				) {
+					await page.getByRole( 'link', { name: 'Dismiss' } ).click();
+				}
+
```

Comment:
A notice on the WPCOM site blocks the test from clicking on the button. If displayed, clicking on "Dismiss" will solve the issue. Please let me know if there's a better way to handle this. 

<img width="960" alt="Screenshot 2024-09-16 at 19 04 16" src="https://github.com/user-attachments/assets/5452ee27-e9f4-460e-81f0-573bb8cae7f4">

Reply:
There is a better way, using `addLocatorHandler`: https://playwright.dev/docs/api/class-page#page-add-locator-handler

Reply:
This is great, Adi. TIL!

Done in [487a8cc](https://github.com/woocommerce/woocommerce/pull/51422/commits/487a8cc37e088adecf8e88881494ba41d9bdec9d). Test is passing against WPCOM and wp-env.